mandatory/4-tax.js

```diff
@@ -17,7 +19,10 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price) {
+  let newPrice = calculateSalesTax(price);
+  return "£" + newPrice.toFixed(2);
```

Comment:
Excellent. Maybe one small improvement? Every time you use the `calculateSalesTax` function, you'll have to call `toFixed` on the result. How can you save from having to do that each time you call it?

Reply:
do not understand question , sorry